mcs/class/System.Web/System.Web.Util/HttpEncoder.cs

```diff
@@ -35,7 +35,7 @@
 using System.Configuration;
 using System.IO;
 using System.Text;
-#if NET_4_0 && !MOBILE
+#if !NO_SYSTEM_WEB_DEPENDENCY && NET_4_0 && !MOBILE
```

Comment:
Yep. I could look at replicating those change to other targets. But I was trying for smallest diff mostly.

Thoughts?